src/pages/settings/Profile/Contacts/ContactMethodDetailsPage.js

```diff
@@ -220,10 +221,14 @@ class ContactMethodDetailsPage extends Component {
      * @param {Boolean} isOpen
      */
     toggleDeleteModal(isOpen) {
-        InteractionManager.runAfterInteractions(() => {
+        if (canUseTouchScreen()) {
```

Comment:
```suggestion
        if (canUseTouchScreen() && isOpen) {
```
This condition can be updated as we don't close any keyboard on closing the modal

Reply:
Done